standard/clause_0_front_material.adoc

```diff
@@ -18,19 +18,19 @@ Given the intended use case of downloading data to a browser, there is no functi
+Implementation experience has shown that CoverageJSON is an effective, efficient format, friendly to web and application developers, and therefore consistent with the current OGC API developments. CoverageJSON supports the efficient transfer from big data stores of useful quantities of data to lightweight clients, such as browsers and mobile applications. This enables local manipulation of the coverage data in a format familiar to, and popular with, web developers, and readily usable, for example, by science researchers.
 
 CoverageJSON can be used to encode coverages and collections of coverages. Coverage data may be gridded or non-gridded, and data values may represent continuous values (such as temperature) or discrete categories (such as land cover classes). CoverageJSON uses JSON-LD to provide interoperability with RDF and Semantic Web applications and to reduce the potential size of the payload.
 
-Relatively large datasets can be handled efficiently in a "web-friendly" way by partitioning information among several CoverageJSON documents, including a tiling mechanism. Nevertheless, CoverageJSON is not intended to be a replacement for efficient binary formats such as NetCDF, HDF or GRIB, and is not intended primarily to store or transfer very large datasets in bulk.
+Relatively large datasets can be handled efficiently in a “web-friendly” way by partitioning information among several CoverageJSON documents, including a tiling mechanism. Nevertheless, CoverageJSON is not intended to be a replacement for efficient binary formats such as NetCDF, HDF or GRIB, and is not intended primarily to store or transfer very large datasets in bulk.
 
-The simplest and most common use case is to embed all the data values of all variables in a Coverage object within the CoverageJSON document, so that it is "self-contained". A self-contained, standalone, document supports the use of very simple clients.
+Embedding the data values of all variables in a Coverage object within the CoverageJSON document is the simplest and most common use case. Such “standalone” document supports the use of very simple clients.
 
-The next-simplest use case is to put data values for each variable (parameter) in separate array objects in separate CoverageJSON documents which are linked from the Coverage object. This is useful for a multi-variable dataset, such as one with temperature, humidity, wind speed, etc., to be recorded in separate files. This allows the client to load only the variables of interest.
+The next simplest use case is to put data values for each variable (parameter) in separate array objects in separate CoverageJSON documents which are linked from the Coverage object. This is useful for a multi-variable dataset, such as one with temperature, humidity, wind speed, etc., to be recorded in separate files. This allows the client to only load the variables of interest.
```

Comment:
I think the hyphen is correct grammatically but don't feel strongly. I prefer the existing version of the last sentence, but it might be even better to say, "This allows the client to select the variables it wishes to load."

Reply:
I will revert the last sentence. Apart from the split-infinitive grammar argument, I think the original had a better emphasis.